app/controllers/api/databases.php

```diff
@@ -1467,6 +1551,87 @@ function updateAttribute(
             ->dynamic($attribute, Response::MODEL_ATTRIBUTE_DATETIME);
     });
 
+App::post('/v1/databases/:databaseId/collections/:collectionId/attributes/relationship')
+    ->alias('/v1/database/collections/:collectionId/attributes/relationship', ['databaseId' => 'default'])
+    ->desc('Create Relationship Attribute')
+    ->groups(['api', 'database'])
+    ->label('event', 'databases.[databaseId].collections.[collectionId].attributes.[attributeId].create')
+    ->label('scope', 'collections.write')
+    ->label('audits.event', 'attribute.create')
+    ->label('audits.resource', 'database/{request.databaseId}/collection/{request.collectionId}')
+    ->label('usage.metric', 'collections.{scope}.requests.update')
+    ->label('usage.params', ['databaseId:{request.databaseId}'])
+    ->label('sdk.namespace', 'databases')
+    ->label('sdk.auth', [APP_AUTH_TYPE_KEY])
+    ->label('sdk.method', 'createRelationshipAttribute')
+    ->label('sdk.description', '/docs/references/databases/create-relationship-attribute.md')
```

Comment:
Where is this file? I don't see it in this PR 👀

Reply:
+1

Reply:
@gewenyu99 Added now...